roles/edpm_libvirt/molecule/default/prepare.yml

```diff
@@ -44,12 +47,63 @@
         owner: root
         group: root
         mode: 0750
+
     - name: open port 22 (edpm_nftables will active this later)
       become: true
-      copy:
+      ansible.builtin.copy:
         dest: /var/lib/edpm-config/firewall/sshd-networks.yaml
         content: |
           - rule_name: 003 Allow SSH
             rule:
               proto: tcp
               dport: 22
+
+    # edpm_nova molecule depends on edpm_libvirt so it tests
+    # the case where this directory does not exsit
+    - name: Create Ceph client file directory
+      become: true
+      ansible.builtin.file:
+        path: '/var/lib/openstack/config/ceph/'
+        state: directory
+        owner: root
+        group: root
+        mode: 0750
```

Comment:
i suspect this mode:750 is the problem
it is likely applying that to 
/var/lib/openstack/config/
disabling traverablity

we had the same issue with /var/lib/libvirt
https://github.com/openstack-k8s-operators/edpm-ansible/commit/e58e59bfd97537e3373083f7899fe20389ee5ce9

we should use "0755" for directories

750 is ok for files.